tests/kani/AsyncAwait/manual_spawn.rs

```diff
@@ -201,3 +290,22 @@ fn arc_spawn_deterministic_test() {
     );
     assert_eq!(x.load(Ordering::Relaxed), 2);
 }
+
+// #[kani::proof]
+// #[kani::unwind(4)]
+// fn nondeterministic_schedule() {
+//     let x = Arc::new(AtomicI64::new(0)); // Surprisingly, Arc verified faster than Rc
+//     let x2 = x.clone();
+//     kani::spawnable_block_on(
+//         async move {
+//             let x3 = x2.clone();
+//             kani::spawn(async move {
+//                 x3.fetch_add(1, Ordering::Relaxed);
```

Comment:
Maybe augment this to cover the scenario where this is run first and run second.

Reply:
I'm not sure what you mean here.

Reply:
File is deleted now anyway, so resolving this.